Sources/AppStoreConnect/Networking/Transport.swift

```diff
@@ -48,9 +48,9 @@ extension URLSession: Transport {
     public func send(request: URLRequest, decoder: JSONDecoder) async throws -> Response<Data> {
         // These depend on swift-corelibs-foundation, which have not implemented the
         // Task-based API for URLSession.
-        #if (os(Linux) || os(Windows)) && swift(<6.0)
+        #if (os(Linux) || os(Windows))
             return try await withCheckedThrowingContinuation { continuation in
-                send(request: request, decoder: decoder, completion: continuation.resume)
+                send(request: request, decoder: decoder) { result in continuation.resume(with: result) }
```

Comment:
The derived closure from `continuation.resume` is not promoted to `Sendable`. But the continuation is `continuation`, thus the inline close capturing `continuation` can be derived as `Sendable`.

Reply:
In addition, the new `asyn` on URLSession in swift 6 seems to crash the run time, maybe it will be fixed in a new version of swift, until that point, full rollback to `withCheckedThrowingContinuation`.